lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -116,17 +116,11 @@ class _FixedAxisArtistHelperBase(_AxisArtistHelperBase):
         lambda self: {"left": (0, 0), "right": (1, 0),
                       "bottom": (0, 0), "top": (0, 1)}[self._loc]))
 
+    @_api.delete_parameter("3.9", "nth_coord")
```

Comment:
Maybe add an addendum that this is inferred from _loc_?

Reply:
Sure.